src/data/people/schema.js

```diff
@@ -6,7 +6,6 @@ const peopleSchema = gql`
 
   extend type Person {
     impersonationParameter: String! @deprecated(reason: "No longer used.")
-    isGroupLeader: Boolean
```

Comment:
If we remove `isGroupLeader` won't the group leader link on the Connect screen no longer work?

Reply:
I moved it to the groups schema

Reply:
Haha, of course. Plus I just could test in the client and make sure it works. 🤦‍♂️